src/libraries/System.Private.CoreLib/src/System/Reflection/CustomAttributeNamedArgument.cs

```diff
@@ -8,41 +8,34 @@ namespace System.Reflection
         public static bool operator ==(CustomAttributeNamedArgument left, CustomAttributeNamedArgument right) => left.Equals(right);
         public static bool operator !=(CustomAttributeNamedArgument left, CustomAttributeNamedArgument right) => !left.Equals(right);
 
-        private readonly MemberInfo m_memberInfo;
-        private readonly CustomAttributeTypedArgument m_value;
+        private readonly MemberInfo _memberInfo;
+        private readonly CustomAttributeTypedArgument _value;
 
         public CustomAttributeNamedArgument(MemberInfo memberInfo, object? value)
         {
             if (memberInfo == null)
                 throw new ArgumentNullException(nameof(memberInfo));
```

Comment:
Could be moved to switch expression bellow

Reply:
I expect that we convert all of these null checks to the new `!!` in near future that would be even better.